### ISSUE TITLE
plotEnrichment with --keepExons throws error

### ISSUE THREAD

**Issue by steffenheyne** [fidelram/deepTools#559]
With that deeptools 2.5.1 call:

```
plotEnrichment -p 8 \
-b STAR_genomic/B6_11wks_1.bam STAR_genomic/B6_11wks_2.bam \
--BED genes.bed \
--plotFile deepTools_qc/plotEnrichment/plotEnrichment.exons.png \
--labels B6_11wks_1 B6_11wks_2 \
--plotTitle 'Fraction of reads in regions (exons only)' \ 
--outRawCounts deepTools_qc/plotEnrichment/plotEnrichment.exons.tsv \
--variableScales --keepExons
```

I get:

```
multiprocessing.pool.RemoteTraceback: 

Traceback (most recent call last):
  File "/package/python-3.6.1/lib/python3.6/multiprocessing/pool.py", line 119, in worker
    result = (True, func(*args, **kwds))
  File "/package/python-3.6.1/lib/python3.6/multiprocessing/pool.py", line 44, in mapstar
    return list(map(*args))
  File "/package/deeptools-2.5.1/lib/python3.6/site-packages/deeptools/plotEnrichment.py", line 279, in getEnrichment_worker
    gtf = Enrichment(args.BED, keepExons=args.keepExons, labels=args.regionLabels)
  File "/package/deeptools-2.5.1/lib/python3.6/site-packages/deeptoolsintervals/enrichment.py", line 203, in __init__
    self.parseBED(fp, line, 12, feature=bname, labelColumn=labelColumn)
  File "/package/deeptools-2.5.1/lib/python3.6/site-packages/deeptoolsintervals/enrichment.py", line 69, in parseBED
    self.parseBEDcore(line, ncols, feature)
  File "/package/deeptools-2.5.1/lib/python3.6/site-packages/deeptoolsintervals/enrichment.py", line 43, in parseBEDcore
    self.tree.addEnrichmentEntry(self.mungeChromosome(cols[0]), x, y, strand, score, feature)
RuntimeError: pyAddEnrichmentEntry received invalid bounds!

"""

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/package/deeptools-2.5.1/bin/plotEnrichment", line 11, in <module>
    main(args)
  File "/package/deeptools-2.5.1/lib/python3.6/site-packages/deeptools/plotEnrichment.py", line 470, in main
    verbose=args.verbose)
  File "/package/deeptools-2.5.1/lib/python3.6/site-packages/deeptools/mapReduce.py", line 142, in mapReduce
    res = pool.map_async(func, TASKS).get(9999999)
  File "/package/python-3.6.1/lib/python3.6/multiprocessing/pool.py", line 608, in get
    raise self._value
RuntimeError: pyAddEnrichmentEntry received invalid bounds!`
```

any idea?

**Comment by dpryan79** [comment]
I didn't remember there even being a `--keepExons` option in `plotEnrichment`, I thought it was keeping everything. Can you try it without that option and see if it still keeps them?

**Comment by dpryan79** [comment]
Gah, looks like I swapped the offset and width column in the BED parsing with `--keepExons`. This only affects plotEnrichment, but I guess this means that you're the first person to try this mixture of settings and files, since the error you got is guaranteed to happen (it was seeing the first offset, 0, as the width, which understandably caused an error). I'll fix this for 2.5.3, which might just come out today...

**Comment by dpryan79** [comment]
This is now fixed in version 2.5.3, which should be available via pip and conda today.
